pkg/webservice/handlers_test.go

```diff
@@ -840,10 +834,11 @@ func TestGetPartitionNodesUtilJSON(t *testing.T) {
 	// setup
 	partition := setup(t, configDefault, 1)
 	appID := "app1"
-	node1ID := "node-1"
-	node2ID := "node-2"
-	node3ID := "node-3"
-
+	const (
```

Comment:
Could you reuse the solution used to fix `TestGetNodeUtilisation`?

Reply:
Okay! I will fix it. THX !